eks/worker_lt.tf

```diff
@@ -92,5 +94,16 @@ resource "aws_launch_template" "node" {
       virtual_name = try(block_device_mappings.value.virtual_name, null)
     }
   }
+
+  dynamic "metadata_options" {
+    for_each = var.metadata_options != {} ? [var.metadata_options] : []
+    content {
+      http_endpoint               = try(metadata_options.value.http_endpoint, "disabled")
+      http_put_response_hop_limit = try(metadata_options.value.http_put_response_hop_limit, 2)
```

Comment:
Should the try fallback to 1 maybe? (if people don't know/need the option because they don't know about it, defaulting to most secure is probably the best option)